python-kubectl/README.md

```diff
@@ -0,0 +1,114 @@
+# Voting App deployed with raw Kubernetes manifests
+
+This example shows how to leverage [Okteto](https://cloud.okteto.com) to develop a python app directly in the cloud.
+This example is deployed using raw Kubernetes manifests.
+
+## Step 1: Install the Okteto CLI
+
+Install the Okteto CLI by running the following command in your local terminal:
+
+MacOS/Linux
+
+```console
+curl https://get.okteto.com -sSfL | sudo sh
+```
+
+Windows
+
+```console
+wget https://downloads.okteto.com/cloud/cli/okteto-Windows-x86_64 -OutFile c:\windows\system32\okteto.exe
+```
+
+## Step 2: Login from the Okteto CLI
+
+```console
+$ okteto login
+```
+
+This command will give you an Okteto Space, where you can create Okteto Environments to code and collaborate.
+Under the hood, an Okteto Space is mapped to a Kubernetes Namespace.
+
+## Step 3: Configure kubectl to point to Okteto
+
+> If you don´t have `kubectl` installed, follow this [guide](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+Okteto gives you `kubectl` credentials configured to work in our multi-tenant cluster. These credentials have network policies, quotas, pod security policies, admission webhooks, roles, role bindings and limit ranges to control what you can do in your Okteto Space. In order to download the credentials, execute:
+
+```console
+$ okteto kubeconfig
+ ✓  Kubeconfig stored at /Users/pablo/.okteto/.kubeconfig
+ i  Configure kubectl to work on your Okteto Space by running:
+    export KUBECONFIG=/Users/pablo/.okteto/.kubeconfig
+```
+
+and now execute (remember to change `pablo` by your username):
+
+```console
+$ export KUBECONFIG=/Users/pablo/.okteto/.kubeconfig
+```
+
+## Step 4: Deploy the Voting App
+
+Clone this repository and move to this example folder.
+
+```console
+git clone https://github.com/okteto/cloud-samples
+cd cloud-samples/python-kubectl
+```
+
+Edit the file `manifests\ingress.yaml` and substitute `pchico83` by your Github ID in lines 9 and 18.
+
+> Okteto only let you create ingress rules for domains ending in `githubid.okteto.net`.
```

Comment:
This is not that clear, I thought for a second that we actually needed the `githubid` word in there. Maybe something like `*-$YOUR_GITHUB_ID.okteto.net` ?